### DISCUSSION TITLE
Local playwright tests not running (`npm run test:e2e`)

### DISCUSSION THREAD

**Discussion by lewisblackburn** [epicweb-dev/epic-stack#841] [Q&A]
I have a fresh install both on Windows and MacOS and have run the `npm run test:e2e` command. The tests all seem to fail. I think this is because the local web server isn't running properly but I'm not sure why.

![image](https://github.com/user-attachments/assets/bf80556c-0a11-450a-8ccb-5fa49409caea)
![image](https://github.com/user-attachments/assets/1e216a85-50b0-4f7d-9ac4-378d203fe135)

**Comment by affeldt28** [comment]
Did you find out why?
I have the same problem. I was able to nail down, that it has something to do with the dev-env. Using `npm run start:mocks` to start the web server instead of `npm run dev` works.
